integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -292,10 +285,15 @@ public void testNotPreexistedOpServiceAccountCreateOperatorNegativeInstall() thr
   }
 
   /**
-   * Negative test : Helm will install the second operator with same target domain namespace as the
-   * first operator.
+   * Install operator usab-1 with target DomainNameSpace [usab-domainns-1].
+   * Install operator usab-2 with same target DomainNamesapce [usab-domainns-1].
+   * Second operator should fail to install with following exception 
```

Comment:
Same comment... I'm less concerned with the fact that Helm failed with a certain message than that the Helm install failed and that the chart was not installed.

Reply:
I removed the Helm version check